Source/WebCore/crypto/cocoa/CryptoAlgorithmEd25519Cocoa.cpp

```diff
@@ -27,12 +27,17 @@
 #include "CryptoAlgorithmEd25519.h"
 
 #include "CryptoKeyOKP.h"
+#if HAVE(SWIFT_CPP_INTEROP)
+#include <pal/PALSwift.h>
+#endif
 #include <pal/spi/cocoa/CoreCryptoSPI.h>
 
 namespace WebCore {
 
-static ExceptionOr<Vector<uint8_t>> signEd25519(const Vector<uint8_t>& sk, size_t, const Vector<uint8_t>& data)
+static ExceptionOr<Vector<uint8_t>> signEd25519(const Vector<uint8_t>& sk, const Vector<uint8_t>& data)
 {
+    if (sk.size() != ed25519KeySize)
```

Comment:
Can we use std::array<uint8_t, ed25519KeySize> instead of Vector?  Same in several places.

Reply:
Soon we will have to add support for Ed448 and then the size will not be fixed. So Vector is probably better ?
I could make it a variant but then all that ifelse just pollutes the code :D